src/App.tsx

```diff
@@ -1,24 +1,161 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { deleteTodo, getTodos, postTodo } from './Api/todos';
+import { TodosList } from './components/TodoList';
+import { FilterTodosBy } from './types/FilterTodosBy/FilterTodoBy';
 
-const USER_ID = 0;
+const USER_ID = 6998;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState('');
+  const [filterBy, setFilterBy] = useState(FilterTodosBy.All);
+  const [query, setQuery] = useState('');
+
+  const todosFromServer = async (userId: number) => {
```

Comment:
it's a name for constant, not for function. Also, use `useCallback` here to not re-create this method on every rerender